go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -368,35 +368,39 @@ func (oc *DefaultNetworkController) Run(ctx context.Context) error {
 
 	// Sync external gateway routes. External gateway may be set in namespaces
 	// or via pods. So execute an individual sync method at startup
-	oc.cleanExGwECMPRoutes()
+	WithWatchDurationMetricNoError("external gateway router", oc.cleanExGwECMPRoutes)
```

Comment:
is this something we are interested in? I guess it doesn't hurt but this isnt really a watcher, it just cleans up stale stuff (sync) function. Also it should be called "external gateway routes", and maybe add the word "cleanup"?

Reply:
Yeah I added this to make we see where the summarized time comes from, about the name - the metric is actually called `sync_duration_seconds` - probably not the best name, but at least it can be applied to watchers and syncs like this one.